_drafts/2023-12-02-teorema-cap.md

```diff
@@ -324,4 +324,8 @@ Em resumo, o teorema CAP é útil para compreensões e discussões iniciais sobr
 
 [Teorema CAP](https://docs.aws.amazon.com/pt_br/whitepapers/latest/availability-and-beyond-improving-resilience/cap-theorem.html)
 
-[Princípios de funcionamento ACID vs BASE nos bancos de dados ](https://edge.uol/en/insights/article/principios-de-funcionamento-acid-vs-base-nos-bancos-de-dados/)
\ No newline at end of file
+[Princípios de funcionamento ACID vs BASE nos bancos de dados ](https://edge.uol/en/insights/article/principios-de-funcionamento-acid-vs-base-nos-bancos-de-dados/)
+
+[Please stop calling databases CP or AP](https://martin.kleppmann.com/2015/05/11/please-stop-calling-databases-cp-or-ap.html)
+
+[Martin Kleppmann. 2015. A Critique of the CAP Theorem](https://arxiv.org/abs/1509.05393)
```

Comment:
Fã clube do Martin Kleppmann marcando presença 😄

Reply:
Lembrei de mais um material dele:
https://github.com/ept/hermitage
https://martin.kleppmann.com/2014/11/25/hermitage-testing-the-i-in-acid.html

Tem também a discussão sobre modelos de consistência do Jepsen, mas é um pouco menos relacionado com CAP